lib/tests/export-all-types.test.ts

```diff
@@ -104,7 +104,7 @@ describe("export-all-types", () => {
         expect(data).toEqual({
             schemas: {
                 Settings: "z.object({ theme_color: z.string(), features: Features.min(1) }).partial().passthrough()",
-                Author: "z.object({ name: z.union([z.string(), z.number()]).nullable(), title: Title.min(1).max(30), id: Id, mail: z.string(), settings: Settings }).partial().passthrough()",
+                Author: "z.object({ name: z.union([z.string().nullable(), z.number()]).nullable(), title: Title.min(1).max(30), id: Id, mail: z.string(), settings: Settings }).partial().passthrough()",
```

Comment:
Up on line 69 you'll see Author's `name` property is set to `nullable: true`, so before this test was asserting union types did _not_ have the chained validation applied to their sub-types